packages/client/src/components/app/blocks/form/FormBlock.svelte

```diff
@@ -0,0 +1,101 @@
+<script>
+  import { getContext } from "svelte"
+  import BlockComponent from "components/BlockComponent.svelte"
+  import Block from "components/Block.svelte"
+  import { makePropSafe as safe } from "@budibase/string-templates"
+  import InnerFormBlock from "./InnerFormBlock.svelte"
+
+  export let actionType
+  export let dataSource
+  export let size
+  export let disabled
+  export let fields
+  export let labelPosition
+  export let title
+  export let showSaveButton
+  export let showDeleteButton
+  export let rowId
+  export let actionUrl
+
+  const { fetchDatasourceSchema } = getContext("sdk")
+
+  let schema
+  let providerId
+  let repeaterId
+
+  $: fetchSchema(dataSource)
```

Comment:
dataSource appears to be an unused parameter

Reply:
It's not a param, but needs to be referenced in the reactive statement so that svelte knows to execute this function again whenever dataSource changes. Making it a param would actually not change the behaviour at all since the function already uses dataSource, which will just reference the latest version from props.

It's a little confusing perhaps, so it may as well be made a parameter for the sake of comprehension.

Reply:
Ah yes I get it, that's good to know!